src/Layout/Navigation/Navigation.tsx

```diff
@@ -51,7 +51,7 @@ export class Navigation extends React.Component<NavigationProps> {
                             <a
                                 className="navbar-link"
                                 target="_blank"
-                                href={wikiUrl?.concat("#Lavinia")}
+                                href={wikiUrl}
```

Comment:
This isn't right. The "Lavinia" page is the one explaining about the application; the Wiki page is more general.